wcfsetup/install/files/lib/acp/form/BoxAddForm.class.php

```diff
@@ -393,6 +393,37 @@ public function readFormParameters()
         if ($this->boxType === 'system') {
             $this->boxController = ObjectTypeCache::getInstance()->getObjectType($this->boxControllerID);
         }
+
+        $this->readConditions();
+    }
+
+    private function readConditions(): void
+    {
+        foreach ($this->groupedConditionObjectTypes as $groupedObjectTypes) {
+            foreach ($groupedObjectTypes as $objectTypes) {
+                if (\is_array($objectTypes)) {
+                    foreach ($objectTypes as $objectType) {
+                        $objectType->getProcessor()->readFormParameters();
+                        if ($objectType->getProcessor() instanceof MultiPageCondition) {
```

Comment:
Should you compare against objectTypeID here, like you do in BoxHandler? You would not need the assertion then, because an ID obviously is unique.